client/src/components/Icons/FAIcon.stories.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+
+
+import FontIcon from './FAIcon'
+
+storiesOf('FontIcon', module)
+    .add('PininterestIcon', () => <FontIcon name={['fab', 'pinterest']}/>)
+    .add('FacebookIcon', () => <FontIcon name={['fab', 'facebook']}/>)
+    .add('InstagramIcon', () => <FontIcon name={['fab', 'instagram']}/>)
```

Comment:
Can you make the titles here proper english? I.e. `Iconts` instead of `FontIcon`, `Pinterest` instead of `PinterestIcon`, `Facebook`, instead of `FacebookIcon`.

Reply:
Also, can you add some non-social icons? There are some from the mock you could add, like a calendar, a magnifying glass, etc